### ISSUE TITLE
can't resolve ngx-stars

### ISSUE THREAD

**Issue by Linknpay** [hughjdavey/ngx-stars#10]
Hi,
Thank you for the updates, but since 2 days i can not build the project anymore : 
```

./src/app/modules/dressing/dressing.module.ts:13:0-43 - Error: Module not found: Error: Can't resolve 'ngx-stars' in '/Users/...
```

**Comment by hughjdavey** [comment]
Can you provide more context? What version of the library? And are you using it via package.json? Perhaps a link to your project or maybe a small project that reproduces the issue?

**Comment by Linknpay** [comment]
When I force it to 1.5.1 in package.json it works well 
`    "ngx-stars": "~1.5.1"`
When I let it install the last version it does not work
`    "ngx-stars": "^1.5.1"`

Now when I try to use it in different modules with the last version, I got an error directly in my IDE that NgxStarsModule can not be found. 
`import {NgxStarsModule} from 'ngx-stars' `

**Comment by hughjdavey** [comment]
Ok thanks for that I was able to reproduce. Will look into it.

**Comment by hughjdavey** [comment]
Should be fixed in 1.6.3 :+1: 

Thanks for pointing it out. Does it also solve the ivy problem in your other issue #8 as I built it with Angular 14?
